packages/flutter_tools/lib/src/compile.dart

```diff
@@ -15,24 +15,38 @@ import 'globals.dart';
 
 typedef void CompilerMessageConsumer(String message);
 
+class CompilerOutput {
+  String outputFilename;
+  int errorCount;
+
```

Comment:
Should the error messages themselves be here too?

Reply:
Perhaps we can add them here later, if it would make sense to expose them like this.
Currently assumption is that complete compiler output is presented to the user which includes errors/warnings/nits.